mdsuite/calculators/einstein_distinct_diffusion_coefficients.py

```diff
@@ -230,29 +230,26 @@ def _post_operation_processes(self, species: Union[str, tuple] = None):
             result = self._fit_einstein_curve([self.time, abs(self.msd_array)])
             properties = {"Property": self.database_group,
                           "Analysis": self.analysis_name,
-                          "Subject": "_".join(species),
+                          "Subject": [species[0], species[1]],
```

Comment:
`"Subject":species` should also work then, I think?

Reply:
Species is a tuple but yes I am sure I could also do something like list(species)

Reply:
Okay so yes one can just write list(species)